staff/sean-daryanani/app-template/about.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About Us</title>
+</head>
+
+<body>
+    <header>
+        <h1>About Us</h1>
+    </header>
+    <main>
+        <section class="about">
+            <article>
+                <header>
+                    <h2>Our Mission</h2>
+                </header>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque deserunt iusto ullam perferendis,
+                    repellat vel neque sint </p>
+            </article>
+            <article>
+                <header>
+                    <h2>About</h2>
+                </header>
+                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Sapiente ullam quae voluptate possimus
+                    nemo molestiae. Sint vero praesentium labore error voluptates harum ipsam velit, facere, obcaecati
+                    quis, eligendi aut tempore.</p>
+            </article>
```

Comment:
may you want to use a section here? sub sections can also be used (section inside section).

Also, a header might not be needed here

Reply:
agree